lib/tasks/amr_importer/import_all.rake

```diff
@@ -6,7 +6,6 @@ namespace :amr do
       begin
         FileUtils.mkdir_p config.local_bucket_path
         Amr::Importer.new(config).import_all
-        Database::VacuumService.new([:amr_data_feed_readings])
```

Comment:
The vacuum service would have been called multiple times for this job on the same table if left here (once for each S3_folder), so, I moved it out of here and to the end, so that it is only called once for this job. Hope this is the right thing to do!